src/sentry/static/sentry/app/views/performance/vitalDetail/utils.tsx

```diff
@@ -1,28 +1,81 @@
 import {Location, Query} from 'history';
 
+import {IconCheckmark, IconFire, IconWarning} from 'app/icons';
 import {Series} from 'app/types/echarts';
 import {getAggregateAlias, WebVital} from 'app/utils/discover/fields';
 import {decodeScalar} from 'app/utils/queryString';
-
-import {WEB_VITAL_DETAILS} from '../transactionVitals/constants';
+import theme from 'app/utils/theme';
 
 export function generateVitalDetailRoute({orgSlug}: {orgSlug: string}): string {
   return `/organizations/${orgSlug}/performance/vitaldetail/`;
 }
 
-export const vitalsThresholdFields = {
-  [WebVital.FCP]: 'count_at_least(measurements.fcp, 3000)',
-  [WebVital.LCP]: 'count_at_least(measurements.lcp, 4000)',
-  [WebVital.FID]: 'count_at_least(measurements.fid, 300)',
-  [WebVital.CLS]: 'count_at_least(measurements.cls, 0.25)',
+export const webVitalPoor = {
+  [WebVital.FP]: 3000,
+  [WebVital.FCP]: 3000,
+  [WebVital.LCP]: 4000,
+  [WebVital.FID]: 300,
+  [WebVital.CLS]: 0.25,
+};
+
+export const webVitalMeh = {
+  [WebVital.FP]: 1000,
+  [WebVital.FCP]: 1000,
+  [WebVital.LCP]: 2500,
+  [WebVital.FID]: 100,
+  [WebVital.CLS]: 0.1,
 };
+
+export const vitalsPoorFields = {
+  [WebVital.FP]: `count_at_least(measurements.fp, 3000)`,
+  [WebVital.FCP]: `count_at_least(measurements.fcp, 3000)`,
+  [WebVital.LCP]: `count_at_least(measurements.lcp, 4000)`,
+  [WebVital.FID]: `count_at_least(measurements.fid, 300)`,
+  [WebVital.CLS]: `count_at_least(measurements.cls, 0.25)`,
+};
+
+export const vitalsMehFields = {
+  [WebVital.FP]: `count_at_least(measurements.fp, 1000)`,
+  [WebVital.FCP]: `count_at_least(measurements.fcp, 1000)`,
+  [WebVital.LCP]: `count_at_least(measurements.lcp, 2500)`,
+  [WebVital.FID]: `count_at_least(measurements.fid, 100)`,
+  [WebVital.CLS]: `count_at_least(measurements.cls, 0.1)`,
```

Comment:
Should we think about adding a new discover function that will help use have smaller requests? Perhaps a function that returns a list with the `[bad, meh, good]` results?

Reply:
I've thought about building a special query for the cards, Will was looking at it.

Reply:
:+1: Totally agreed, Shift some of these calls with nearly 20 fields to special endpoints will help avoid other issues we're having too

Reply:
We can and should ship this. A specific endpoint would be ideal, and would let us optimize loading data and ensure vitals always look the same in each place they show up.